vm/src/exceptions.rs

```diff
@@ -84,29 +82,29 @@ impl PyBaseException {
     }
 
     #[pyproperty(name = "__traceback__", setter)]
-    fn set_traceback(&self, traceback: Option<PyTracebackRef>, vm: &VirtualMachine) -> PyResult {
+    fn setter_traceback(&self, traceback: Option<PyTracebackRef>, vm: &VirtualMachine) -> PyResult {
```

Comment:
it looks a little bit awkward that getter has `get_` prefixed name but setter has `setter_` prefixed names

Reply:
Yeah I agree, but it was conflicting with the public setter methods so I just renamed them to something that seemed appropriate. I might go with `__traceback__/set____traceback__`, but that might be too many underscores : smiley:

Reply:
How about adapting #1647 for these methods? This will not fit for everytime but fortunately `set_traceback` and `setter_traceback` can share the implementation in this case.

Reply:
That sounds good, it would be nice if we could omit the `&VirtualMachine` parameter too as it's not really necessary, but I'm not sure if that's doable.

Reply:
would you check #1649 for this?